Patch by Steve Kemp <skx@debian.org> for dsniff >= 2.4b1, which adds the
missing OpenSSL includes for header files.

--- dsniff-2.4b1/sshcrypto.c		2006-11-02 23:41:11.000000000 -0300
+++ dsniff-2.4b1/sshcrypto.c.sshcrypto	2006-11-02 23:41:55.000000000 -0300
@@ -14,6 +14,8 @@
 
 #include <sys/types.h>
 #include <openssl/ssl.h>
+#include <openssl/blowfish.h>
+#include <openssl/des.h>
 
 #include <err.h>
 #include <stdio.h>
